src/useFormState.js

```diff
@@ -109,7 +110,7 @@ export default function useFormState(initialState, options) {
 
     function validate(
       e,
-      value = isRaw ? undefined : e.target.value,
+      value = isRaw ? e : e.target.value,
```

Comment:
Why is `e` not `undefined` here when called from `onBlur(e)`?  I'd think this would depend on the custom control passing the value to `onBlur()`?  (That's why I did this: ﻿https://github.com/wsmd/react-use-form-state/pull/78/files#diff-0dd3e7ebb317129e9d204bec747da593R112 instead before I realized you'd already opened a PR.  :P)

Reply:
I also wanna add to @jwalton's comment - the `value` argument represents what's currently in the `formState`, not what the input passes back which is `e`.

This line makes both arguments identical. Although it won't matter _most of the time_, please be aware that users of this library can also change the value stored in the form-state on raw inputs by returning a custom value from the `onChange` option.

From the docs:

> You can also provide an onChange option with a return value in order to map the value passed from the custom control's onChange to a different value in the form state.

Therefore, I think it's important to make this distinction between the different arguments we're exposing on `L120`:

```
validate(value, values, e)
         ─────  ──────  ─
           │      │     │
           │      │     └ The event itself or the raw value obtained directly
           │      │       from the input's onChange or onBlur handlers
           │      │
           │      └ The formState's values (values of all inputs)
           │
           └ The value of the input (stored) in the formState's values
```

This is was implemented as intended in @jwalton link above.